tests/test_symbolics.py

```diff
@@ -705,7 +708,7 @@ def test_multibounds_op(self):
         op.apply(time_M=5)
         fnorm2 = norm(f)
 
-        assert fnorm == fnorm2
+        assert math.isclose(fnorm, fnorm2, rel_tol=1e-8)
```

Comment:
should use `np.isclose`

Reply:
>np.isclose returns a boolean array

only for arrays, it returns a numpy boolean for scalars which those are

Reply:
So it does